### ISSUE TITLE
Pylance Import Issue 

### ISSUE THREAD

**Issue by heshangambhir** [microsoft/pyright#4664]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Screenshots or Code**
If applicable, add screenshots or the text of the code (surrounded by triple back ticks) to help explain your problem.
```python
def foo(self) -> str:
    return 3
```

If your code relies on symbols that are imported from a third-party library, include the associated import statements and specify which versions of those libraries you have installed.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool? Which version? You can find the version of the VS Code extension by clicking on the Pyright icon in the extensions panel.

**Additional context**

**Comment by erictraut** [comment]
Thanks for submitting a bug report, but we'll need more information than that. Perhaps this was submitted in error because it looks like you haven't filled in the bug report template. I'm going to close it. If you have a legitimate bug to report, please provide the requested information, and we'd be happy to help.

**Comment by heshangambhir** [comment]
hey sorry for this I was trying to report a bug but before I could write anything It got submitted I open a new bug report
